src/server/server_family.cc

```diff
@@ -47,6 +48,8 @@ using namespace std;
 ABSL_FLAG(string, dir, "", "working directory");
 ABSL_FLAG(string, dbfilename, "dump", "the filename to save/load the DB");
 ABSL_FLAG(string, requirepass, "", "password for AUTH authentication");
+ABSL_FLAG(string, save_schedule, "", "glob spec for the time to save a snapshot which matches HH:MM 24h time");
+ABSL_FLAG(bool, save_schedule_use_utc, false, "use UTC when specifying the time to save a snapshot");
```

Comment:
I think we can omit `save_schedule_use_utc`  - if I am not mistaken cloud environment is usually aligned with UTC (i.e. default images in AWS, GCP), hence local time is the UTC time. lets wait for someone provide a concrete use-case that would require this flag.